checker/src/org/checkerframework/checker/initialization/InitializationVisitor.java

```diff
@@ -181,6 +184,19 @@ protected boolean checkContract(
                             return true;
                         }
                     }
+                } else if (fa.getReceiver() instanceof LocalVariable) {
+                    // TODO: what is the best way to handle other types of
+                    // receivers?
+                    Element elem = ((LocalVariable) fa.getReceiver()).getElement();
+                    AnnotatedTypeMirror recvType = atypeFactory.getAnnotatedType(elem);
```

Comment:
The store might not contain anything for the receiver and then I would need to do the fallback myself.
The factory will look into the store and use it's value for me, so in case something refined is available, it should will used.
Is there a method that would allow me to go from `fa.getReceiver()` to an AnnotatedTypeMirror, without me having to know it's a local variable, getting the Element, and then asking the factory?